pytorch_lightning/accelerators/ddp_cpu_spawn_backend.py

```diff
@@ -186,6 +188,19 @@ def early_stopping_should_stop(self, pl_module):
         should_stop = stop == self.trainer.world_size
         return should_stop
 
+    def set_world_ranks(self, process_idx):
+        self.trainer.local_rank = process_idx
+        self.trainer.global_rank = self.trainer.node_rank * self.trainer.num_processes + process_idx
+        self.trainer.world_size = self.trainer.num_nodes * self.trainer.num_processes
+
+    def model_to_device(self, model, process_idx):
+        # in ddp cpu we don't actually move models to a device
```

Comment:
we should explicitly move them to cpu here, since we don't know on which device it was initially.

Reply:
good call